locations/spiders/isola_tesori_it.py

```diff
@@ -0,0 +1,40 @@
+from locations.categories import Categories, apply_category
+from locations.hours import CLOSED_IT, DAYS_IT, NAMED_DAY_RANGES_IT, NAMED_TIMES_IT, OpeningHours
+from locations.json_blob_spider import JSONBlobSpider
+
+
+def clean_strings(iterator):
+    return list(filter(bool, map(str.lower, map(str.strip, iterator))))
+
+
+class IsolaTesoriITSpider(JSONBlobSpider):
+    name = "isola_tesori_it"
+    item_attributes = {"brand": "L'Isola dei Tesori", "brand_wikidata": "Q108578003"}
+    start_urls = ["https://www.isoladeitesori.it/on/demandware.store/Sites-idt-it-Site/it_IT/Stores-FindStores"]
+    locations_key = "stores"
+
+    def post_process_item(self, item, response, location):
+        item["extras"]["addr:province"] = item.pop("state")
```

Comment:
I'm not sure I get what you mean... In Italy there are provinces and regions, not states, and the keywords of the DictParser map either provinces or regions to states (depending on the source). To not mess with all the other countries' behaviour, that needs to be remapped...

Reply:
Well it only parses Saturday and Sunday at the moment.

Reply:
We don't map to OSM perfectly. Our address hierarchy is a bit different.

Reply:
I get that, I was always looking at the OSM perspective - maybe it could be useful to explain that a bit on a doc file? (or maybe I missed it if it's already existing)